assets/javascripts/kitten/components/cards/engagement-card/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Marger } from '../../../components/layout/marger'
+import { Text } from '../../../components/typography/text'
+import { pxToRem } from '../../../helpers/utils/typography'
+import { ScreenConfig } from '../../../constants/screen-config'
+
+const StyledCard = styled.div`
+  border: solid transparent;
+  border-radius: ${pxToRem(15)};
+  position: relative;
+  overflow: hidden;
+  width: ${pxToRem(102)};
+  height: ${pxToRem(130)};
+  background-image: url(${({ imageSrc }) => imageSrc});
+  background-repeat: no-repeat;
+  cursor: pointer;
+
+  @media (min-width: ${pxToRem(ScreenConfig.M.min)}) {
+    width: ${pxToRem(111)};
+    height: ${pxToRem(150)};
+  }
+`
+
+const StyledText = styled(Text)`
+  position: absolute;
+  text-align: center;
+  bottom 5%;
```

Comment:
```suggestion
  bottom: 5%;
```

Il faudrait peut-être rajouter `left` et `right` (pour avoir des marges)